.github/workflows/ci.yml

```diff
@@ -60,3 +75,32 @@ jobs:
         run: rustup update stable && rustup default stable
       - name: rustfmt
         run: ./ci/rustfmt.sh
+
+  # These jobs don't actually test anything, but they're used to tell bors the
+  # build completed, as there is no practical way to detect when a workflow is
+  # successful listening to webhooks only.
+  #
+  # ALL THE PREVIOUS JOBS NEEDS TO BE ADDED TO THE `needs` SECTION OF THIS JOB!
```

Comment:
This is a realistic way for bors to receive build status from Github Actions, which do a lot of jobs (It's used in [`rust-lang/rust` etc.](https://github.com/rust-lang/rust/blob/62da38d00d8d09dbaaa092c7b5e7ea343fdc2126/src/ci/github-actions/ci.yml#L631-L633), [previously we had to manually manage all job names.](https://github.com/crossbeam-rs/crossbeam/pull/518/files#diff-2f8c9db9253b64f9a332c09f492cd54e))